pkg/collect/mysql.go

```diff
@@ -20,6 +20,8 @@ func Mysql(c *Collector, databaseCollector *troubleshootv1beta2.Database) (Colle
 	} else {
 		query := `select version()`
 		row := db.QueryRow(query)
+		defer db.Close()
```

Comment:
@eochoa260 - move this before the `QueryRow` to ensure its hooked up incase something happens